packages/theme/src/components/dashboard/posts/PostEditor/index.vue

```diff
@@ -137,7 +145,7 @@ function hideTitleError(event: FormFieldErrorType) {
 }
 
 async function updatePostHandler() {
-  if (!post.title.trim()) {
+  if (!post.title.trim() && post.title.length <= maxTitleLength) {
```

Comment:
There seems to be inconsistent behaviour.

https://github.com/user-attachments/assets/8c001e07-086e-4d6b-babe-3a21a4e70e56